src/classify/adaptive.h

```diff
@@ -69,49 +63,44 @@ typedef struct
   LIST TempProtos;
   ADAPTED_CONFIG Config[MAX_NUM_CONFIGS];
 } ADAPT_CLASS_STRUCT;
-typedef ADAPT_CLASS_STRUCT *ADAPT_CLASS;
+using ADAPT_CLASS = ADAPT_CLASS_STRUCT*;
 
-typedef struct
-{
+typedef struct {
```

Comment:
`typedef struct` is a C thing.
Probably we could replace them with just `struct`.

Reply:
The main focus of this PR is `modernize-use-using`. Yes, those `typedef struct` statements should be replaced by C++ `struct` or `class` statements where possible (not in `capi.h`), That's one of the many things left for future commits.